src/components/UI/MarkdownPreview.vue

```diff
@@ -1,15 +1,24 @@
 <template>
   <span
+    v-if="props"
```

Comment:
すみません、コメントのコードが間違っていました🙏 
"`inline`が指定されたとき"なので↓のようになります！
```suggestion
    v-if="props.inline"
```